<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="frameRange_layout"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="options_layout">
     <item>
      <layout class="QVBoxLayout" name="options2_layout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QCheckBox" name="solveFocalLength_checkBox">
         <property name="text">
          <string>Solve Focal Length</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="solveLensDistortion_checkBox">
         <property name="text">
          <string>Solve Lens Distortion</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="options1_layout">
       <item>
        <layout class="QHBoxLayout" name="sceneGraphMode_layout">
         <item>
          <widget class="QLabel" name="sceneGraphMode_label">
           <property name="text">
            <string>Scene Graph</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sceneGraphMode_comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="evalComplexGraphs_checkBox">
         <property name="text">
          <string>Evaluate Mesh Rivets</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
